android/app/build.gradle

```diff
@@ -0,0 +1,66 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 22
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "org.freedesktop.gstreamer.play"
+        minSdkVersion 10
+        targetSdkVersion 22
```

Comment:
What is the difference between minSdkVersion and targetSdkVersion? Will the result run on android-9/10 or will it require setting the targetSdkVersion to those first?